File: tests/fixtures/browser-worker-workspace/dashboards/worker-runtime/transforms/scaled.js
async function transform(context) {
  const delay = Number(context.compute_params.delay_ms || 0);
  await new Promise(resolve => setTimeout(resolve, delay));
  return {
    main: context.inputs.rows.map(row => ({
      name: row.name,
      value: Number(row.value) * 10,
    })),
  };
}

File: tests/fixtures/browser-python-analysis-workspace/dashboards/browser-python/transforms/scaled.py
def transform(context):
    assert context.query_inputs["batch"] == 3
    rows = context.inputs["rows"]
    factor = context.compute_params["factor"]
    return {
        "main": [
            {"name": row["name"], "value": row["value"] * factor}
            for row in rows
        ]
    }
